mesheryctl/internal/cli/root/mesh/validate.go

```diff
@@ -41,7 +41,7 @@ mesheryctl mesh validate --adapter [name of the adapter] --tokenPath [path to to
 
 ! Refer below image link for usage
 * Usage of mesheryctl mesh validate
-# ![mesh-validate-usage](../../../../docs/assets/img/mesheryctl/mesh-validate.png)
+# ![mesh-validate-usage](https://docs.meshery.io/assets/img/mesheryctl/mesh-validate.png)
```

Comment:
Can't we replace it with something the links below.

```suggestion
# ![mesh-validate-usage](./assets/img/mesheryctl/mesh-validate.png)
```

The reason being that if someone wants to change this image and they run a local version of docs, currently it will still point to the image in `docs.meshery.io` but replacing it will a relative path will point to the image present in the system where the server is being run.

Reply:
This relative reference makes sense to me. It's almost always better to use a relative reference than an absolute one.